src/legacy/server/saved_objects/import/import_saved_objects.ts

```diff
@@ -41,19 +43,26 @@ export async function importSavedObjects({
   overwrite,
   savedObjectsClient,
 }: ImportSavedObjectsOptions): Promise<ImportResponse> {
-  const objectsToImport = await collectSavedObjects(readStream, objectLimit);
+  let objectsToImport = await collectSavedObjects(readStream, objectLimit);
+
+  let errors: ImportError[] = [];
+  ({ filteredObjects: objectsToImport, errors } = await validateReferences(
```

Comment:
I _personally_ think combining mutable bindings and destructuring is kind of confusing, might I suggest:
```ts
const { filteredObjects, errors } = await validateReferences(
  await collectSavedObjects(readStream, objectLimit),
  savedObjectsClient
);
```